.travis.yml

```diff
@@ -9,15 +9,12 @@ matrix:
         # https://github.com/travis-ci/travis-ci/issues/8363#issuecomment-355090242
         - python: "2.7"
           env: TOXENV=py27
+        - python: "3.4"
+          env: TOXENV=py34
```

Comment:
I'm not entirely sure we need this env anymore @lukpueh any thoughts as to why this sneaked here?

Reply:
This was just the result of a mis-merge, which I have already fixed. @lukpueh Don't worry about it, as I have already gotten rid of the Python 3.4 env in the updated .travis.yml on my fork.